lib/extensions/ar_miq_set.rb

```diff
@@ -35,44 +37,59 @@ def sets
   def make_memberof(set)
     set.add_member(self)
   end
+
+  def make_not_memberof(set)
```

Comment:
this is used in only 1 place. @Fryguy can we just remove this method?

Reply:
I'd be ok with that... Doesn't necessarily have to be in this PR, but if that makes things cleaner, 👍 from me.

Reply:
agreed. ignore this nick. (unless you don't wanna ;) )

Reply:
Leaving this here as a reminder, but no, I have no intention of doing this in this PR.  To much refactoring and moving parts already happening that this can be done as a follow up where it can be focused and easier to review.